src/components/PostsWrap.jsx

```diff
@@ -7,16 +7,24 @@ import GridDisplay from './GridDisplay.jsx';
 import ListDisplay from './ListDisplay.jsx';
 import { fetchRecommendations } from '../lib/database';
 
+const activeLinkColumns = {
+  popular: { orderBy: 'total_stars' },
+  upvoted: { orderBy: 'votes' },
+  discussed: { orderBy: 'issues' },
+  recent: { orderBy: 'avg_recency_score' },
+};
+
```

Comment:
This is so much cleaner and testable. As far as testing strategy, we should move towards using [data/hot.json](https://github.com/open-sauced/hot/blob/main/src/data/hot.json) as mock data for testing.